src/index.ts

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* BEGIN RG ADDITION */
+export * from "./colours";
```

Comment:
This is necessary as there's lots of places `ScratchBlocks.Colours` is referenced:

(one example)
https://github.com/scratchfoundation/scratch-gui/blob/develop/src/lib/blocks.js#L70